packages/common/package.json

```diff
@@ -34,7 +34,7 @@
     "react-currency-input-field": "^3.6.0",
     "react-hook-form": "^7.22.0",
     "react-i18next": "^11.15.1",
-    "react-query": "^3.34.7",
+    "react-query": "^3.34.12",
```

Comment:
I bumped this as there was an issue with inlined selector functions. Thought this might be a problem but it wasn't. Kept the bump anyway.